src/components/grouped-character/grouped-character-test.stories.tsx

```diff
@@ -77,3 +78,30 @@ NewValidation.args = {
   separator: "-",
   ...getCommonTextboxArgs(),
 };
+
+export const GroupedCharacterComponent = ({
+  onChange,
+  ...props
+}: {
+  onChange?: (modifiedEvent: any) => void;
+}) => {
```

Comment:
Since we already have a type interface for `GroupCharacter`'s props, doing the following will correctly type the `onChange` prop for you.

This will also type the other remaining props, since currently `GroupCharacterComponent` is typed to only expect a `onChange` prop, when we could be passing the other `GroupCharacter` props as well.

```suggestion
}: GroupCharacterProps) => {
```